scripts/tests/test_zkai_d64_rmsnorm_to_projection_bridge_input.py

```
@@ -0,0 +1,66 @@
+        self.assertNotEqual(payload["projection_input_row_commitment"], BRIDGE.OUTPUT_ACTIVATION_COMMITMENT)
+
+    def test_payload_rejects_projection_full_output_relabeling(self) -> None:
+        payload = BRIDGE.build_payload()
+        payload["projection_input_row_commitment"] = BRIDGE.OUTPUT_ACTIVATION_COMMITMENT
+        with self.assertRaisesRegex(BRIDGE.BridgeInputError, "relabeled as full output"):
+            BRIDGE.validate_payload(payload)
+
+    def test_payload_rejects_bridge_row_equality_drift(self) -> None:
+        payload = BRIDGE.build_payload()
+        payload["rows"][0]["projection_input_q8"] += 1
+        with self.assertRaisesRegex(BRIDGE.BridgeInputError, "bridge row equality drift"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)